lua/nvim-treesitter/parsers.lua

```diff
@@ -1277,7 +1277,7 @@ list.sql = {
   install_info = {
     url = "https://github.com/derekstride/tree-sitter-sql",
     files = { "src/parser.c" },
-    requires_generate_from_grammar = true,
+    branch = "gh-pages",
```

Comment:
> Not an issue, you just need to edit parsers.lua to remove the generate_from_grammar key.

Is this all that's needed then?

Reply:
Should be, if the lockfile hash points to the head of that branch

Reply:
Which you need to do manually in this case.

Reply:
I updated the lockfile to point to the revision from the `gh-pages` branch.